Gemfile

```diff
@@ -4,13 +4,15 @@ gem 'rake'
 gem 'yard'
 
 group :test do
-  gem 'coveralls', :require => false
-  gem 'rspec'
-  gem 'rspec-autotest'
-  gem 'autotest-standalone'
+  gem 'activesupport'
```

Comment:
yeah, it's needed for `deep_dup`, which was needed to make the tests work after i `freeze`d everything